flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/AppendStreamScan.scala

```diff
@@ -36,20 +36,20 @@ import org.apache.flink.table.typeutils.TimeIndicatorTypeInfo
   * It ensures that types without deterministic field order (e.g. POJOs) are not part of
   * the plan translation.
   */
-class DataStreamScan(
+class AppendStreamScan(
```

Comment:
Shouldn't this belong to the previous commit?

Reply:
Yes, sorry about this.